client-gateway/bot/driver.py

```
@@ -1,32 +1,377 @@
     BTN_DRIVER = buttons['BTN_DRIVER']
     BTN_MY_ORDERS = buttons['BTN_MY_ORDERS']
+    BTN_REGISTER_DRIVER = buttons['BTN_REGISTER_DRIVER']
+    BTN_GO_ONLINE = buttons['BTN_GO_ONLINE']
+    BTN_GO_OFFLINE = buttons['BTN_GO_OFFLINE']
+    BTN_DRIVER_STATUS = buttons['BTN_DRIVER_STATUS']
+    
+    driver_menu_unregistered = keyboards['driver_menu_unregistered']
+    driver_menu_registered = keyboards['driver_menu_registered']
     
-    driver_menu = keyboards['driver_menu']
+    safe_send = helpers['safe_send']
```

Flake8 (7.3.0)

[error] local variable 'safe_send' is assigned to but never used

(F841)